pkg/web/api/v1/transaction.go

```diff
@@ -21,21 +21,21 @@ import (
 //===========================================================================
 
 type Transaction struct {
-	ID                 uuid.UUID `json:"id"`
-	Source             string    `json:"source"`
-	Status             string    `json:"status"`
-	Counterparty       string    `json:"counterparty"`
-	CounterpartyID     ulid.ULID `json:"counterparty_id,omitempty"`
-	Originator         string    `json:"originator,omitempty"`
-	OriginatorAddress  string    `json:"orginator_address,omitempty"`
-	Beneficiary        string    `json:"beneficiary,omitempty"`
-	BeneficiaryAddress string    `json:"beneficiary_address,omitempty"`
-	VirtualAsset       string    `json:"virtual_asset"`
-	Amount             float64   `json:"amount"`
-	LastUpdate         time.Time `json:"last_update,omitempty"`
-	EnvelopeCount      int64     `json:"envelope_count,omitempty"`
-	Created            time.Time `json:"created"`
-	Modified           time.Time `json:"modified"`
+	ID                 uuid.UUID  `json:"id"`
+	Source             string     `json:"source"`
+	Status             string     `json:"status"`
+	Counterparty       string     `json:"counterparty"`
+	CounterpartyID     ulid.ULID  `json:"counterparty_id,omitempty"`
+	Originator         string     `json:"originator,omitempty"`
+	OriginatorAddress  string     `json:"orginator_address,omitempty"`
+	Beneficiary        string     `json:"beneficiary,omitempty"`
+	BeneficiaryAddress string     `json:"beneficiary_address,omitempty"`
+	VirtualAsset       string     `json:"virtual_asset"`
+	Amount             float64    `json:"amount"`
+	LastUpdate         *time.Time `json:"last_update,omitempty"`
```

Comment:
This will set the `LastUpdate` field to `nil` if it is empty and return nothing to the front-end to prevent "null last update" problems.